packages/react-atlas-core/webpack.config.js

```diff
@@ -11,6 +11,21 @@ module.exports = {
     publicPath: '/lib/',
     libraryTarget: 'commonjs2'
   },
+  externals: [{
```

Comment:
I run into multiple copies of react when I wrapped bs table library, maybe the cause is it has peer dependency on react, after using externals the error disappear.
The externals Also can short bundled index 2.2MB -> 1.1MB in my branch without minifying, and webpack mentioned it's useful for library developer so maybe we could consider this approach.

Once we use external it could avoid the same issue if we want to use other library in the future.